app/controllers/concerns/sessions_controller_override.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module SessionsControllerOverride
+  extend ActiveSupport::Concern
+
+  included do
+    after_action :csv_auto_verify, only: :create
+  end
+
+  def csv_auto_verify
+    return unless current_user.organization.available_authorizations.include?("csv_census")
+
+    record = Decidim::Verifications::CsvDatum.find_or_create_by(
+      email: current_user.email,
+      organization: current_user.organization
+    )
+    return unless record
```

Comment:
```suggestion
    return unless Decidim::Verifications::CsvDatum.exists?(
      email: current_user.email,
      organization: current_user.organization
    )
```
e don't want to create a record here!

Reply:
Done